lib/ansible/galaxy/dependency_resolution/dataclasses.py

```diff
@@ -537,7 +541,11 @@ def canonical_package_id(self):
 
     @property
     def is_virtual(self):
-        return self.is_scm or self.is_subdirs
+        return self.is_scm or self.is_subdirs or self.is_ansible
+
+    @property
+    def is_ansible(self):
```

Comment:
I feel like this name needs a bit more accuracy.
```suggestion
    def is_ansible_core_runtime(self):
```

Reply:
I changed it to `is_ansible_runtime` in the last commit.